package.json

```diff
@@ -11,7 +11,7 @@
     "format": "prettier . --list-different",
     "format:ci": "prettier . --check --cache",
     "lint:js": "eslint \"./**/*.{js,cjs,mjs}\"",
-    "lint:md": "markdownlint \"**/*.md\" --ignore node_modules",
+    "lint:md": "markdownlint \"**/*.md\" --ignore node_modules --disable MD041 MD013",
```

Comment:
`MD041` = Starting a .md with the h1 tag. Not necessary for this app since some elements are in layouts.
`MD013` = line length rule (80 chars). Not interested in keeping this.